lib/core.yaml

```diff
@@ -0,0 +1,28 @@
+linter:
```

Comment:
As a drive by comment, we should have yaml comments here (and in `recommend.yaml`) describing what the files are, how to use them, where the format docs are, ...

We could do that as part of this PR (or, track the work in an issue in this repo).

Reply:
It's a good point.  I think we want some thoughtful docs in general (here, README, etc).  Maybe we can land this (and #1) and iterate so we can start experimenting?

Tracking issue for docs: https://github.com/dart-lang/lints/issues/5